src/invertObject.js

```diff
@@ -19,7 +19,17 @@
  * @return {Object}
  */
 function invertObject(items) {
-  // write code here
+  const invertedItems = {};
+
+  for (const [key, value] of Object.entries(items)) {
```

Comment:
Можно перед объектом создать `const entries = Object.entries(item)`, а тут просто
```suggestion
  for (const [key, value] of entries) {
```
Улучшит читаемость